assets/views/home/references.ejs

```diff
@@ -19,39 +19,13 @@
           {{ok.approveAll.event ? "" : "Approve all"}}
         </button>
       <td>
-        <span ng-if="user.isMod && refUser">{{<%= theUser %>.references.filter(isEvent).filter(isApproved).length}}</span>
+        <span ng-if="user.isMod && !onIndexPage">{{<%= theUser %>.references.filter(isEvent).filter(isApproved).length}}</span>
       </td>
       <td class="hidden-sm hidden-xs"></td>
     </tr>
     <tbody id="collapseevents" class="collapse">
     <tr ng-repeat="reference in <%= theUser %>.references | filter: isEvent | orderBy: 'createdAt'">
-      <td><strong>{{$index + 1}}.</strong></td>
-      <td><a ng-href="{{reference.url}}">{{reference.gave}} for {{reference.got}}</a></td>
-      <td class="hidden-sm hidden-xs">
-        <a ng-href="{{getRedditUser(reference.user2)}}">
-          {{getRedditUser(reference.user2)}}
-        </a>
-      </td>
-        <td>
-            <input type="checkbox" ng-change="approve(reference.id, reference.approved)"
-                   ng-model="reference.approved" ng-if="user.isMod && refUser"/>
-            <span ng-if="user.isMod && refUser && reference.edited"><font class="editstar" title="This reference was edited at some point.">*</font></span>
-            <span ng-if="user.isMod && refUser && reference.verified"><sup><font class="greencheck" title="This trade has been approved and logged by both users.">✓</font></sup></span>
-        </td>
-      <td>
-        <a href="#viewReferenceModal" data-toggle="modal" ng-click="editReference(reference)" class="icon">
-          <span class="glyphicon glyphicon-info-sign"></span>
-        </a>
-        <a href="#editReferenceModal" data-toggle="modal" ng-click="editReference(reference)"
-           ng-if="(!refUser || (user.name === refUser.name))" class="icon">
-          <span class="glyphicon glyphicon-edit"></span>
-        </a>
-        <a href="#" ng-really-switch="deleteRef" class="icon"
-           ng-really-click="deleteRef(reference.id)"
-           ng-if="(!refUser || (user.name === refUser.name) || user.isMod)">
-          <span class="glyphicon glyphicon-trash"></span>
-        </a>
-      </td>
+      <%- partial('row.ejs') %>
```

Comment:
I got rid of some of the repetition in `references.ejs`, now rows for trades are abstracted out.